README.md

```diff
@@ -62,79 +43,97 @@ example your `config/settings.html` or other configuration driven items
 - config/settings.html
 ```
 
-Download all the theme files
 
-```
-theme download
-```
+#### Here is a Breakdown of the Fields:
 
-Upload a theme file
+`api_key`
+The API key generated in your private app.
 
-```
-theme upload assets/layout.liquid
-```
+`password`
+The password generated in your private app.
 
-Remove a theme file
+`store`
+The address of your store (note there is no leading http:// or https://)
 
-```
-theme remove assets/layout.liquid
-```
+`theme_id`
+The theme id of the theme that should be responding to commands. If no theme id is specified, the currently active theme will be modified.
 
-Completely replace shop theme assets with the local assets
+`whitelist_files`
+Specifies which files and directories should be responding to changes. If nothing is specified the  `assets/, config/, layout/, snippets/, templates/ and locales/` directories will be modified according to commands entered.
 
-```
-theme replace
-```
+`ignore_files`
+Specifies which files should be explicitly ignored and not affected by any commands entered
 
-Watch the theme directory and upload any files as they change
 
-```
-theme watch
-```
+## Commands
```

Comment:
I like this though I think it looks a little awkward because some things don't fit on a single line.

Perhaps try something like?

`command`
-----------------------
Details about the command 

`other command`
-----------
Details about this other command